### ISSUE TITLE
Cypress Test

### ISSUE THREAD

**Issue by Andrey-1992** [Andrey-1992/suspicious-ip-ui#5]
As a developer, I should be able to make test suites with cypress for the dashboard for multi-pages in the app: 
- Home view
- Local IP
- Searched IP
- Saved IPs
